smack-core/src/main/java/org/jivesoftware/smack/util/ParserUtils.java

```diff
@@ -194,4 +198,17 @@ public static URI getUriFromNextText(XmlPullParser parser) throws XmlPullParserE
         return uri;
     }
 
+    public static Map<String, String> getCustomAttributes(XmlPullParser parser) {
+        Map<String, String> customAttributes = new LinkedHashMap<>();
+        List<String> standardAttributes = Arrays.asList(new String[] {"xml:lang", "id", "to", "from", "type"});
+            for (int i = 0; i < parser.getAttributeCount(); i++) {
+                String attributeName = parser.getAttributeName(i);
+                boolean isLangAttribute = "lang".equals(attributeName) && "xml".equals(parser.getAttributePrefix(i));
```

Comment:
I reused the logic I found in `PacketParserUtils.getLanguageAttribute()`. Do you think it's overkill in this case ?